adodb.inc.php

```diff
@@ -1519,7 +1510,7 @@ function _Execute($sql,$inputarr=false) {
 			}
 			$this->_queryID = _adodb_debug_execute($this, $sql,$inputarr);
 		} else {
-			$this->_queryID = @$this->_query($sql,$inputarr);
+			$this->_queryID = @$this->Query($sql,$inputarr);
```

Comment:
Same comment as for connect() - query() is the public interface function, _query() is the driver-specific